.github/workflows/csv-to-csvw.yml

```diff
@@ -52,52 +53,68 @@ jobs:
             file_name="${file_path##*/}"
             file_extension="${file##*.}"
             out_path="out/${file_path}/"
-            
+            root_path=$(echo "$file_path" | cut -d "/" -f1)
+
             echo "file_path: ${file_path}"
             echo "file_name: ${file_name}"
             echo "file_extension: ${file_extension}"
             echo "out_path: ${out_path}"
+            echo "root_path: ${root_path}"
 
-            if [[ $file_extension == "csv" ]]; then
+            if [[ ($file_extension == "csv" || $file_extension == "json") && $root_path != "out" ]]; then
               echo $'\n'
               echo "Processing file ${file}"
               echo "Output path is ${out_path}"
 
-              csv_file=${file}
-              for file_secondary in "${added_modified_files[@]}"; do
-                file_secondary_name="${file_secondary%.*}"
-                file_secondary_extension="${file_secondary##*.}"
-                if [[ "${file_secondary_name}.${file_secondary_extension}" == "${file_path}.json" ]]; then
-                  config_file="${file_secondary_name}.json"
+              csv_file=""
+              config_file=""
+              if [[ $file_extension == "csv" ]]; then
+                csv_file=${file}
+                for file_secondary in "${added_modified_files[@]}"; do
+                  file_secondary_name="${file_secondary%.*}"
+                  file_secondary_extension="${file_secondary##*.}"
+                  if [[ "${file_secondary_name}.${file_secondary_extension}" == "${file_path}.json" ]]; then
+                    config_file="${file_secondary_name}.json"
+                  fi
+                done
+              elif [[ $file_extension == "json" ]]; then
+                config_file=${file}
+                if [[ -f "${file_path}/${file_name}.csv" ]]; then
```

Comment:
So you're looking for a file called `some_file.json.csv`? We should reallly be looking for a file called `some_file.csv`

Reply:
Once the input is detected as a json, this piece of code looks for the csv file. For example, if the input file is `some_file.json`, then this code looks for `some_file.csv`. More specifically, when the input is `some_file.json`, the `file_name` variable is `some_file`.

Reply:
But isn't `$file_name` `some_file.json` as this point - so you're just adding `.csv` on to the end of it?

Reply:
@robons No, the `$file_name` is going to be `some_file` at this point. E.g. below are the echos:
`
file_name: csv_with_config
file_extension: json
`